### PR TITLE
Update rules and metrics docs

### PR DESCRIPTION
**Pull request by incendial**
### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix
[ ] New rule
[ ] Changes an existing rule
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

File: doc/rules/component_annotation_arguments_ordering.md [hunk 1]
```diff
@@ -26,9 +26,14 @@ The value for `order` may be an array consisting of the following strings (defau
 ### Config example
 
 ```yaml
-component-annotation-arguments-ordering:
-  order:
-    - selector
-    - templates
-    - change-detection
+
```
Comment (dkrutskikh) [inline doc/rules/component_annotation_arguments_ordering.md:29]: @incendial please remove extra blank line

File: doc/rules/no_empty_block.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
 
 ### Example
+
 Bad:
+
 ```dart
+  // LINT
   if ( ... ) {
-
+
```
Comment (dkrutskikh) [inline doc/rules/no_empty_block.md:20]: @incendial please remove trailing space

**Review by dkrutskikh — CHANGES_REQUESTED** [review]

File: doc/rules/component_annotation_arguments_ordering.md (hunk 1 above)
Comment (incendial) [reply doc/rules/component_annotation_arguments_ordering.md]: Done.

File: doc/rules/no_empty_block.md (hunk 2 above)
Comment (incendial) [reply doc/rules/no_empty_block.md]: Done.

**Pull request merged**